### PR TITLE
Add suffixes to typos from "bi" to "bu"

### PR DESCRIPTION
**Pull request by int-y1**
Is `builtin` or `built-in` better? For now, I didn't include typos involving `built(-)in`.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7795,11 +7880,27 @@ bridg->bridge
 brievely->briefly
 brievety->brevity
 brigde->bridge
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:7882]: Tried to find occurrences of `brigade` typos, without much success.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -7795,11 +7880,27 @@ bridg->bridge
...
+brigdes->bridges
 brige->bridge
 briges->bridges
+brigh->bright
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:7886]: Perhaps `brig`? Keys **G** and **H** are neighbours.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -7645,9 +7715,14 @@ bouces->bounces
 boucing->bouncing
 boudaries->boundaries
 boudary->boundary
+bouded->bounded
 bouding->bounding
+boudn->bound
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:7720]: I got 103000 hits for `"boudin sausage"` on Google, but 0 hits for `"boudn sausage"`. I will leave out this typo.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:7882]: There are occurrences of `fire brigde`->`fire brigade`. I will add this.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:7886]: I couldn't find this.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**